blender/arm/logicnode/transform/LN_rotate_object_around_axis.py

```diff
@@ -6,16 +6,29 @@ class RotateObjectAroundAxisNode(ArmLogicTreeNode):
     bl_label = 'Rotate Object Around Axis'
     bl_description = 'Rotate Object Around Axis (Depreciated: use "Rotate Object")'
     bl_icon = 'ERROR'
+    arm_version=2
 
     def init(self, context):
+        super().init(context)
+        print("init() getting called")
```

Comment:
Debug print?

Reply:
...yes.
I thought I removed all of them, but yeah.

Reply:
done